yarn.lock

```diff
@@ -2257,6 +2257,11 @@ glob-parent@^3.1.0, glob-parent@^5.1.2, glob-parent@~5.1.2:
   dependencies:
     is-glob "^4.0.1"
 
+glob-to-regexp@^0.4.1:
```

Comment:
I don't see braces version change here. Can you re-run `yarn`?

Reply:
done.